packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.js

```diff
@@ -39,75 +35,115 @@ const StyledCardContent = styled(() => ({
   alignItems: 'center',
 }))(CardContent);
 
+const UserHeader = styled(({ theme }) => ({
+  alignItems: 'center',
+  marginBottom: theme.sizing.baseUnit * 1.5,
+}))(View);
+
 const StyledBodyText = styled(() => ({
   textAlign: 'center',
 }))(BodyText);
 
-const BodyTextView = styled(({ theme }) => ({
-  marginTop: theme.sizing.baseUnit * 1.5,
-  height: theme.sizing.baseUnit * 16,
-  alignItems: 'center',
-}))(View);
-
 class PrayerCard extends PureComponent {
   handleShowActionSheet = () => {
     this.ActionSheet.show();
   };
 
   render() {
-    const { imageSource, name, text, id, flagRequest, source } = this.props;
+    const {
+      interactive,
+      help,
+      header,
+      avatarSource,
+      avatarSize,
+      created,
+      name,
+      campus,
+      prayer,
+      options,
+    } = this.props;
+
+    // add a cancel button
+    const buttons = options || [];
+    buttons.push({ title: 'Cancel', method: null, destructive: false });
 
-    const cancelIndex = 1;
-    const destructiveIndex = 0;
     const handleOnPress = (index) => {
-      if (index !== cancelIndex) {
-        return flagRequest(id);
-      }
+      const methods = buttons.map((option) => option.method);
```

Comment:
what this be better titled, `buttonMethods`?

Reply:
yep agreed